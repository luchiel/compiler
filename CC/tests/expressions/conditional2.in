5 + (a ? b : c),
5 + a ? b : c