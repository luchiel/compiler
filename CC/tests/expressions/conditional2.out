{,}
| 
+-{+}
| | 
| +-{int 5}
| | 
| +-{?:}
|   | 
|   +-{a}
|   | 
|   +-{b}
|   | 
|   +-{c}
| 
+-{,}
  | 
  +-{?:}
  | | 
  | +-{+}
  | | | 
  | | +-{int 5}
  | | | 
  | | +-{a}
  | | 
  | +-{b}
  | | 
  | +-{c}
  | 
  +-{?:}
    | 
    +-{a}
    | 
    +-{?:}
    | | 
    | +-{aa}
    | | 
    | +-{bb}
    | | 
    | +-{cc}
    | 
    +-{c}
